dom.bs

```diff
@@ -2872,6 +2872,135 @@ before a <var>child</var>, with an optional <i>suppress observers flag</i>, run
 </ol>
 
 
+<p><a lt="Other applicable specifications">Specifications</a> may define <dfn export
+id=concept-node-move-ext>moving steps</dfn> for all or some <a for=/>nodes</a>. The algorithm is
```

Comment:
The DOM standard doesn't wrap on all spaces. Phrasing-level elements need to be kept on a single line. (This applies several times in this PR.)

It also seems okay to drop the explicit ID here as we stopped doing those.